boot/bootdata/livecd.inf

```diff
@@ -48,7 +48,7 @@ HKLM,"SYSTEM\CurrentControlSet\Control\Session Manager\Environment","ALLUSERSPRO
 HKLM,"SYSTEM\CurrentControlSet\Control\Session Manager\Memory Management","PagingFiles",0x00000000,""
 
 ; Font Substitution
-HKLM,"SOFTWARE\Microsoft\Windows NT\CurrentVersion\FontSubstitutes","Arial",0x00000000,"Liberation Sans"
+HKLM,"SOFTWARE\Microsoft\Windows NT\CurrentVersion\FontSubstitutes","Arial",0x00000000,"Arial"
```

Comment:
If you have fonts named "Arial" from the beginning, why is this substitution line still needed? I think it could be removed altogether?
+ Same remark for the corresponding lines in muifonts.h.

Reply:
The user language is changeable. If these registry values are not overwritten, then the old substitutes will be remained.

Reply:
There is no deleting-substitute mechanism.

Reply:
Deleted some Arial substs.